File: client/iih_stub/iih.ci_stub/iih.ci.pre/dto/d/EmergPreDTO.cs
using System;
using xap.mw.core.data;
using xap.mw.coreitf.d;

namespace iih.ci.pre.dto.d
{
    /// <summary>
    /// EmergPreDTO 的摘要说明。
    /// </summary>
    public class EmergPreDTO : BaseDTO {

        public EmergPreDTO() {
 
        }

        /// <summary>
        /// 预检id
        /// </summary>
		public string Id_obspre {
            get { return getAttrVal<string>("Id_obspre",null); }
            set { setAttrVal<string>("Id_obspre", value); }
        }

        /// <summary>
        /// 就诊id
        /// </summary>
		public string Id_ent {
            get { return getAttrVal<string>("Id_ent",null); }
            set { setAttrVal<string>("Id_ent", value); }
        }

        /// <summary>
        /// 就诊类型
        /// </summary>
		public string Code_entp {
            get { return getAttrVal<string>("Code_entp",null); }
            set { setAttrVal<string>("Code_entp", value); }
        }

        /// <summary>
        /// 患者id
        /// </summary>
		public string Id_pat {
            get { return getAttrVal<string>("Id_pat",null); }
            set { setAttrVal<string>("Id_pat", value); }
        }

        /// <summary>
        /// 患者姓名
        /// </summary>
		public string Name_pat {
            get { return getAttrVal<string>("Name_pat",null); }
            set { setAttrVal<string>("Name_pat", value); }
        }

        /// <summary>
        /// 病情等级
        /// </summary>
		public string Id_level_dise {
            get { return getAttrVal<string>("Id_level_dise",null); }
            set { setAttrVal<string>("Id_level_dise", value); }
        }

        /// <summary>
        /// 病情等级编码
        /// </summary>
		public string Sd_level_dise {
            get { return getAttrVal<string>("Sd_level_dise",null); }
            set { setAttrVal<string>("Sd_level_dise", value); }
        }

        /// <summary>
        /// 到院时间
        /// </summary>
		public DateTime? Dt_entry {
            get { return getAttrVal<FDateTime>("Dt_entry",null); }
            set { setAttrVal<FDateTime>("Dt_entry", value); }
        }

        /// <summary>
        /// 到院方式
        /// </summary>
		public string Id_come_way {
            get { return getAttrVal<string>("Id_come_way",null); }
            set { setAttrVal<string>("Id_come_way", value); }
        }

        /// <summary>
        /// 到院方式编码
        /// </summary>
		public string Sd_come_way {
            get { return getAttrVal<string>("Sd_come_way",null); }
            set { setAttrVal<string>("Sd_come_way", value); }
        }

        /// <summary>
        /// 到院方式名称
        /// </summary>
		public string Name_come_way {
            get { return getAttrVal<string>("Name_come_way",null); }
            set { setAttrVal<string>("Name_come_way", value); }
        }

        /// <summary>
        /// 抢救开始时间
        /// </summary>
		public DateTime? Dt_rescue_b {
            get { return getAttrVal<FDateTime>("Dt_rescue_b",null); }
            set { setAttrVal<FDateTime>("Dt_rescue_b", value); }
        }

        /// <summary>
        /// 抢救结束时间
        /// </summary>
		public DateTime? Dt_rescue_e {
            get { return getAttrVal<FDateTime>("Dt_rescue_e",null); }
            set { setAttrVal<FDateTime>("Dt_rescue_e", value); }
        }

        /// <summary>
        /// 陪伴人员id
        /// </summary>
		public string Ids_companion {
            get { return getAttrVal<string>("Ids_companion",null); }
            set { setAttrVal<string>("Ids_companion", value); }
        }

        /// <summary>
        /// 陪伴人员
        /// </summary>
		public string Sds_companion {
            get { return getAttrVal<string>("Sds_companion",null); }
            set { setAttrVal<string>("Sds_companion", value); }
        }

        /// <summary>
        /// 询问流行病史标志
        /// </summary>
		public bool? Fg_chk_eqidemic {
            get { return getAttrVal<FBoolean>("Fg_chk_eqidemic",null); }
            set { setAttrVal<FBoolean>("Fg_chk_eqidemic", value); }
        }

        /// <summary>
        /// 有24时发热
        /// </summary>
		public bool? Fg_has_hot {
            get { return getAttrVal<FBoolean>("Fg_has_hot",null); }
            set { setAttrVal<FBoolean>("Fg_has_hot", value); }
        }

        /// <summary>
        /// 有3天发热
        /// </summary>
		public bool? Fg_has_hot2 {
            get { return getAttrVal<FBoolean>("Fg_has_hot2",null); }
            set { setAttrVal<FBoolean>("Fg_has_hot2", value); }
        }

        /// <summary>
        /// 有最近疫区旅游
        /// </summary>
		public bool? Fg_has_eqidarea {
            get { return getAttrVal<FBoolean>("Fg_has_eqidarea",null); }
            set { setAttrVal<FBoolean>("Fg_has_eqidarea", value); }
        }

        /// <summary>
        /// 有接触动物
        /// </summary>
		public bool? Fg_has_touchanim {
            get { return getAttrVal<FBoolean>("Fg_has_touchanim",null); }
            set { setAttrVal<FBoolean>("Fg_has_touchanim", value); }
        }

        /// <summary>
        /// 询问补充
        /// </summary>
		public string Chk_note {
            get { return getAttrVal<string>("Chk_note",null); }
            set { setAttrVal<string>("Chk_note", value); }
        }

        /// <summary>
        /// 预检服务项目集合
        /// </summary>
		public FArrayList Srvarray {
            get { return getAttrVal<FArrayList>("Srvarray",null); }
            set { setAttrVal<FArrayList>("Srvarray", value); }
        }

        /// <summary>
        /// 复诊
        /// </summary>
		public bool? Fg_revisit {
            get { return getAttrVal<FBoolean>("Fg_revisit",null); }
            set { setAttrVal<FBoolean>("Fg_revisit", value); }
        }
        /// <summary>
        /// 返回属性列表
        /// </summary>
        /// <returns></returns>
        public override string[] getAttrNames()
        {
            return new string[]{ "Id_obspre", "Id_ent", "Code_entp", "Id_pat", "Name_pat", "Id_level_dise", "Sd_level_dise", "Dt_entry", "Id_come_way", "Sd_come_way", "Name_come_way", "Dt_rescue_b", "Dt_rescue_e", "Ids_companion", "Sds_companion", "Fg_chk_eqidemic", "Fg_has_hot", "Fg_has_hot2", "Fg_has_eqidarea", "Fg_has_touchanim", "Chk_note", "Srvarray", "Fg_revisit"};
        }
        
        /// <summary>
        /// 返回全路径DO类名
        /// </summary>
        /// <returns></returns>
        public override string getFullClassName()
        {
            return "iih.ci.pre.dto.d.EmergPreDTO";
        }
    }
}

File: server/iih_pub/iih.ci_pub/src/main/java/iih/ci/pre/dto/d/EmergPreDTO.java
package iih.ci.pre.dto.d;

import xap.mw.core.data.*;
import xap.mw.coreitf.d.*;
import java.math.BigDecimal;

/**
 * 急诊预检DTO DTO数据 
 * 
 */
public class EmergPreDTO extends BaseDTO {
	private static final long serialVersionUID = 1L;
	
	/**
	 * 预检id
	 * @return String
	 */
	public String getId_obspre() {
		return ((String) getAttrVal("Id_obspre"));
	}
	/**
	 * 预检id
	 * @param Id_obspre
	 */
	public void setId_obspre(String Id_obspre) {
		setAttrVal("Id_obspre", Id_obspre);
	}
	/**
	 * 就诊id
	 * @return String
	 */
	public String getId_ent() {
		return ((String) getAttrVal("Id_ent"));
	}
	/**
	 * 就诊id
	 * @param Id_ent
	 */
	public void setId_ent(String Id_ent) {
		setAttrVal("Id_ent", Id_ent);
	}
	/**
	 * 就诊类型
	 * @return String
	 */
	public String getCode_entp() {
		return ((String) getAttrVal("Code_entp"));
	}
	/**
	 * 就诊类型
	 * @param Code_entp
	 */
	public void setCode_entp(String Code_entp) {
		setAttrVal("Code_entp", Code_entp);
	}
	/**
	 * 患者id
	 * @return String
	 */
	public String getId_pat() {
		return ((String) getAttrVal("Id_pat"));
	}
	/**
	 * 患者id
	 * @param Id_pat
	 */
	public void setId_pat(String Id_pat) {
		setAttrVal("Id_pat", Id_pat);
	}
	/**
	 * 患者姓名
	 * @return String
	 */
	public String getName_pat() {
		return ((String) getAttrVal("Name_pat"));
	}
	/**
	 * 患者姓名
	 * @param Name_pat
	 */
	public void setName_pat(String Name_pat) {
		setAttrVal("Name_pat", Name_pat);
	}
	/**
	 * 病情等级
	 * @return String
	 */
	public String getId_level_dise() {
		return ((String) getAttrVal("Id_level_dise"));
	}
	/**
	 * 病情等级
	 * @param Id_level_dise
	 */
	public void setId_level_dise(String Id_level_dise) {
		setAttrVal("Id_level_dise", Id_level_dise);
	}
	/**
	 * 病情等级编码
	 * @return String
	 */
	public String getSd_level_dise() {
		return ((String) getAttrVal("Sd_level_dise"));
	}
	/**
	 * 病情等级编码
	 * @param Sd_level_dise
	 */
	public void setSd_level_dise(String Sd_level_dise) {
		setAttrVal("Sd_level_dise", Sd_level_dise);
	}
	/**
	 * 到院时间
	 * @return FDateTime
	 */
	public FDateTime getDt_entry() {
		return ((FDateTime) getAttrVal("Dt_entry"));
	}
	/**
	 * 到院时间
	 * @param Dt_entry
	 */
	public void setDt_entry(FDateTime Dt_entry) {
		setAttrVal("Dt_entry", Dt_entry);
	}
	/**
	 * 到院方式
	 * @return String
	 */
	public String getId_come_way() {
		return ((String) getAttrVal("Id_come_way"));
	}
	/**
	 * 到院方式
	 * @param Id_come_way
	 */
	public void setId_come_way(String Id_come_way) {
		setAttrVal("Id_come_way", Id_come_way);
	}
	/**
	 * 到院方式编码
	 * @return String
	 */
	public String getSd_come_way() {
		return ((String) getAttrVal("Sd_come_way"));
	}
	/**
	 * 到院方式编码
	 * @param Sd_come_way
	 */
	public void setSd_come_way(String Sd_come_way) {
		setAttrVal("Sd_come_way", Sd_come_way);
	}
	/**
	 * 到院方式名称
	 * @return String
	 */
	public String getName_come_way() {
		return ((String) getAttrVal("Name_come_way"));
	}
	/**
	 * 到院方式名称
	 * @param Name_come_way
	 */
	public void setName_come_way(String Name_come_way) {
		setAttrVal("Name_come_way", Name_come_way);
	}
	/**
	 * 抢救开始时间
	 * @return FDateTime
	 */
	public FDateTime getDt_rescue_b() {
		return ((FDateTime) getAttrVal("Dt_rescue_b"));
	}
	/**
	 * 抢救开始时间
	 * @param Dt_rescue_b
	 */
	public void setDt_rescue_b(FDateTime Dt_rescue_b) {
		setAttrVal("Dt_rescue_b", Dt_rescue_b);
	}
	/**
	 * 抢救结束时间
	 * @return FDateTime
	 */
	public FDateTime getDt_rescue_e() {
		return ((FDateTime) getAttrVal("Dt_rescue_e"));
	}
	/**
	 * 抢救结束时间
	 * @param Dt_rescue_e
	 */
	public void setDt_rescue_e(FDateTime Dt_rescue_e) {
		setAttrVal("Dt_rescue_e", Dt_rescue_e);
	}
	/**
	 * 陪伴人员id
	 * @return String
	 */
	public String getIds_companion() {
		return ((String) getAttrVal("Ids_companion"));
	}
	/**
	 * 陪伴人员id
	 * @param Ids_companion
	 */
	public void setIds_companion(String Ids_companion) {
		setAttrVal("Ids_companion", Ids_companion);
	}
	/**
	 * 陪伴人员
	 * @return String
	 */
	public String getSds_companion() {
		return ((String) getAttrVal("Sds_companion"));
	}
	/**
	 * 陪伴人员
	 * @param Sds_companion
	 */
	public void setSds_companion(String Sds_companion) {
		setAttrVal("Sds_companion", Sds_companion);
	}
	/**
	 * 询问流行病史标志
	 * @return FBoolean
	 */
	public FBoolean getFg_chk_eqidemic() {
		return ((FBoolean) getAttrVal("Fg_chk_eqidemic"));
	}
	/**
	 * 询问流行病史标志
	 * @param Fg_chk_eqidemic
	 */
	public void setFg_chk_eqidemic(FBoolean Fg_chk_eqidemic) {
		setAttrVal("Fg_chk_eqidemic", Fg_chk_eqidemic);
	}
	/**
	 * 有24时发热
	 * @return FBoolean
	 */
	public FBoolean getFg_has_hot() {
		return ((FBoolean) getAttrVal("Fg_has_hot"));
	}
	/**
	 * 有24时发热
	 * @param Fg_has_hot
	 */
	public void setFg_has_hot(FBoolean Fg_has_hot) {
		setAttrVal("Fg_has_hot", Fg_has_hot);
	}
	/**
	 * 有3天发热
	 * @return FBoolean
	 */
	public FBoolean getFg_has_hot2() {
		return ((FBoolean) getAttrVal("Fg_has_hot2"));
	}
	/**
	 * 有3天发热
	 * @param Fg_has_hot2
	 */
	public void setFg_has_hot2(FBoolean Fg_has_hot2) {
		setAttrVal("Fg_has_hot2", Fg_has_hot2);
	}
	/**
	 * 有最近疫区旅游
	 * @return FBoolean
	 */
	public FBoolean getFg_has_eqidarea() {
		return ((FBoolean) getAttrVal("Fg_has_eqidarea"));
	}
	/**
	 * 有最近疫区旅游
	 * @param Fg_has_eqidarea
	 */
	public void setFg_has_eqidarea(FBoolean Fg_has_eqidarea) {
		setAttrVal("Fg_has_eqidarea", Fg_has_eqidarea);
	}
	/**
	 * 有接触动物
	 * @return FBoolean
	 */
	public FBoolean getFg_has_touchanim() {
		return ((FBoolean) getAttrVal("Fg_has_touchanim"));
	}
	/**
	 * 有接触动物
	 * @param Fg_has_touchanim
	 */
	public void setFg_has_touchanim(FBoolean Fg_has_touchanim) {
		setAttrVal("Fg_has_touchanim", Fg_has_touchanim);
	}
	/**
	 * 询问补充
	 * @return String
	 */
	public String getChk_note() {
		return ((String) getAttrVal("Chk_note"));
	}
	/**
	 * 询问补充
	 * @param Chk_note
	 */
	public void setChk_note(String Chk_note) {
		setAttrVal("Chk_note", Chk_note);
	}
	/**
	 * 预检服务项目集合
	 * @return FArrayList
	 */
	public FArrayList getSrvarray() {
		return ((FArrayList) getAttrVal("Srvarray"));
	}
	/**
	 * 预检服务项目集合
	 * @param Srvarray
	 */
	public void setSrvarray(FArrayList Srvarray) {
		setAttrVal("Srvarray", Srvarray);
	}
	/**
	 * 复诊
	 * @return FBoolean
	 */
	public FBoolean getFg_revisit() {
		return ((FBoolean) getAttrVal("Fg_revisit"));
	}
	/**
	 * 复诊
	 * @param Fg_revisit
	 */
	public void setFg_revisit(FBoolean Fg_revisit) {
		setAttrVal("Fg_revisit", Fg_revisit);
	}
}